### PR TITLE
add full-featured-non-premium demo to docs

### PR DESCRIPTION
**Pull request by tylerkelly13**

### CODE REVIEW

File: _includes/codepens/open-source-plugins/index.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    { title: 'New list with dates', description: 'New List with dates', content: '<div class="mceTmpl"><span class="cdate">cdate</span><br /><span class="mdate">mdate</span><h2>My List</h2><ul><li></li><li></li></ul></div>' }
+  ],
+  template_cdate_format: '[Date Created (CDATE): %m/%d/%Y : %H:%M:%S]',
+  template_mdate_format: '[Date Modified (MDATE): %m/%d/%Y : %H:%M:%S]',
+  height: 600,
+  image_caption: true,
+  quickbars_selection_toolbar: 'bold italic | quicklink h2 h3 blockquote quickimage quicktable',
+  noneditable_noneditable_class: "mceNonEditable",
+  toolbar_drawer: 'sliding',
+  spellchecker_dialog: true,
```
Comment (lnewson) [inline _includes/codepens/open-source-plugins/index.js]: These are premium spellchecker settings. Also on that note, it looks like the community spellchecker plugin is missing from this demo?

Comment (lnewson) [reply _includes/codepens/open-source-plugins/index.js]: You'd also have to implement a dummy spellchecker service if you do add the community spellchecker: https://www.tiny.cloud/docs/plugins/spellchecker/#spellchecker_callback

Comment (tylerkelly13) [reply _includes/codepens/open-source-plugins/index.js]: :(

Comment (tylerkelly13) [reply _includes/codepens/open-source-plugins/index.js]: Removing spellchecker from demo, adding to the list of excluded plugins

File: demo/full-featured.md [hunk 2]
```diff
@@ -19,4 +21,18 @@ The following plugins are excluded from this example:
...
+This example includes only non-premium plugins. These plugins are also used in the [Full Featured: Including Premium Plugins Demo above](#fullfeaturedincludingpremiumplugins).
+
+{% include codepen.html id="open-source-plugins" %}
+
+The following plugins are excluded from this example:
+* [All premium plugins](https://www.tiny.cloud/pricing/#demo-enterprise).
+* [Autoresize]({{site.baseurl}}/plugins/autoresize) — Resizes the editor to fit the content.
+* [BBCode]({{site.baseurl}}/plugins/bbcode) — Changes the markup used for the content.
+* [Full Page]({{site.baseurl}}/plugins/fullpage) — Used for modifying HTML `<head>` elements.
+* [Spellchecker]({{site.baseurl}}/plugins/spellchecker) — Requires [Server-Side components]({{site.baseurl}}/enterprise/server/).
```
Comment (lnewson) [inline demo/full-featured.md]: This probably needs to link to https://www.tiny.cloud/docs/general-configuration-guide/spell-checking/, as that link is for the premium components.

File: _includes/codepens/open-source-plugins/index.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    { title: 'New list with dates', description: 'New List with dates', content: '<div class="mceTmpl"><span class="cdate">cdate</span><br /><span class="mdate">mdate</span><h2>My List</h2><ul><li></li><li></li></ul></div>' }
+  ],
+  template_cdate_format: '[Date Created (CDATE): %m/%d/%Y : %H:%M:%S]',
+  template_mdate_format: '[Date Modified (MDATE): %m/%d/%Y : %H:%M:%S]',
+  height: 600,
+  image_caption: true,
+  quickbars_selection_toolbar: 'bold italic | quicklink h2 h3 blockquote quickimage quicktable',
+  noneditable_noneditable_class: "mceNonEditable",
+  toolbar_drawer: 'sliding',
+  content_style: ".mymention{ color: green; }",
```
Comment (lnewson) [inline _includes/codepens/open-source-plugins/index.js]: This is for mentions, so probably shouldn't be here.

File: demo/full-featured.md (hunk 2 above)
Comment (tylerkelly13) [reply demo/full-featured.md]: updated

File: _includes/codepens/open-source-plugins/index.js (hunk 3 above)
Comment (tylerkelly13) [reply _includes/codepens/open-source-plugins/index.js]: removed

**Pull request merged**